cmd/version/cmd_test.go

```diff
@@ -18,38 +20,60 @@ func TestVersionCommand(t *testing.T) {
 }
 
 var _ = Describe("Run Command", func() {
+	var cmd *cobra.Command
+	var r *rosa.Runtime
 
 	var delegateInvokeCount int
-	var buf *bytes.Buffer
 
 	BeforeEach(func() {
-		buf = new(bytes.Buffer)
-		writer = buf
 		delegateInvokeCount = 0
 		delegateCommand = func(cmd *cobra.Command, args []string) {
 			delegateInvokeCount++
 		}
+
+		cmd = makeCmd()
+		initFlags(cmd)
+
+		r = rosa.NewRuntime()
+		DeferCleanup(r.Cleanup)
+	})
+
+	It("It only prints version and build information", func() {
+		args.clientOnly = true
+
+		stdout, stderr, err := test.RunWithOutputCapture(runWithRuntime, r, cmd)
+
+		Expect(err).To(BeNil())
+		Expect(stderr).To(Equal(""))
+		Expect(stdout).To(Equal(fmt.Sprintf("%s (Build: %s)\n", info.Version, info.Build)))
```

Comment:
This could be simplified:
```suggestion
		Expect(stdout).To(ContainSubstring("%s (Build: %s)\n", info.Version, info.Build))
```

Reply:
This depends in my opinion, it could potentially be another message that also contains that substring. Using the equal matcher would enforce that it is exactly the one you are testing for

Reply:
I tend to agree w/ @gdbranco but did change it to ContainsSubstring. For example, in another test using Equal showed that there was an unnecessary tab at the end of a message. A minor nitpick but it was caught during my test writing. :D

Reply:
@gdbranco @thomasmckay Yep. You are right: if there is the risk that the message can be contained in another message and, in that case, the test should fail, then Equal should be used.

Reply:
I'm good leaving it as ContainsSubstring here. I'll keep this discussion in mind for future tests. Very useful!